Links.html

```diff
@@ -9,36 +9,37 @@
         #useful-links a {color: #8E443D;}
         </style>
 </head>
-<ul>
-    <li><a href="index.html">Home</a></li>
-    <li><a href="Glossary.html">Glossary</a></li>
-    <li><a href="Links.html">Links</a></li></ul>
-    <title>Alison's CSD120 Website</title>
-    <meta charset="utf-8">
-</head>
 <body>
+    <header>
+        <nav>
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="Glossary.html">Glossary</a></li>
+                <li><a href="Links.html">Links</a></li>
+            </ul>
+        </nav>
+    </header>
```

Comment:
I'm guessing this was a botched merge.  Be careful to select all the correct lines from the conflicts and also pay attention to the order of lines when you fix things up.  Make sure everything is still valid after you've merged.